modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -228,12 +233,13 @@ export default class AttributeTransitionManager {
 
     if (!buffer) {
       buffer = new Buffer(this.gl, {
-        size,
         data: new Float32Array(toLength),
         usage: GL.DYNAMIC_COPY
       });
     }
 
+    transition.attributeInTransition.update({isGeneric: false, buffer});
```

Comment:
Note: in luma v6: I am have replaced the term "generic" with "constant" in the lower level API, would like to do it on the `Attribute` level as well. Because of the coupling between luma and deck Attributes I held back for now.

Reply:
Just flip the switch before we enter beta? `isGeneric` is used throughout the code.